src/pages/partners/affiliate-ib/_dmt5-cards.js

```diff
@@ -0,0 +1,337 @@
+        marginBottom: '16px',
+    }
+    const header_style = {
+        padding: '14px 0 17px',
+        borderRadius: '0',
+        height: 'auto',
+        boxShadow: 'unset',
+        borderBottom: 'unset',
+        borderTop: '1px solid var(--color-grey-21)',
+    }
+    const StyledButton = styled(Button)`
+        border: none;
+        color: var(--color-red);
+        font-size: 14px;
+        background: none;
+
+        @media ${device.tabletL} {
+            font-size: 14px;
+        }
+    `
+
+    const [is_calculated, setCalculated] = React.useState(false)
+    const toggleCalculated = () => {
+        setCalculated(!is_calculated)
+    }
+
+    return (
+        <StyledCard padding="2.4rem 2.4rem 0">
+            {!is_calculated ? (
+                <>
+                    <CardWrappers>
+                        <Header as="h4" type="sub-section-title" mb="0.8rem">
+                            {data.name}
+                        </Header>
+                        <CardText>{data.description}</CardText>
+                        <AccordionWrapper>
+                            <Accordion has_single_state>
+                                {data.type.map((value, index) => (
+                                    <AccordionItem
+                                        key={index}
+                                        header={value.title}
+                                        parent_style={parent_style}
+                                        style={item_style}
+                                        header_style={header_style}
+                                        plus
+                                    >
+                                        <TableWrapper grid_col_number={2} is_balance={true}>
+                                            {value.assets.map((listedValue, indexValue) => (
+                                                <TC
+                                                    grid_area={'area' + indexValue}
+                                                    key={indexValue}
+                                                >
+                                                    <StyledTrap
+                                                        isTitle="true"
+                                                        headerHeight={value.headerHeight}
+                                                    >
+                                                        <StyledText size="14px" weight="bold">
+                                                            {listedValue.title}
+                                                        </StyledText>
+                                                    </StyledTrap>
+                                                    {listedValue.list.map((data, indexData) => (
+                                                        <TRAPREVERSE
+                                                            even={indexData % 2 ? 'true' : ''}
```

Comment:
can use utility function isEven

Reply:
This is only available on our Crowdin branch currently 😓

Reply:
I cannot found 'isEven function' in utility..Maybe because of the base..

Reply:
got it